### DISCUSSION TITLE
CLA and survey form issue

### DISCUSSION THREAD

**Discussion by ShivamMadlani** [oppia/oppia-android#4295] [Q&A (Other)] [answered]
Hello all,
I signed the CLA and filled the survey form a long time ago dated 4th March 2022. However I did not get any further instructions and was not assigned any mentors. I also opened Two PR's and one issue. But due to some issue from your side, my PR's were closed and not considered. I frequently kept posting this issue on Gitter channel but no solution was proposed there.
Please take a look at it as soon as possible as this is causing me a lot of trouble in working with Oppia. I am also applying for GSoc at oppia this year but this issue is preventing me to have a profound base for the org.
I request all the mentors to please find a way and help me merge my PR's as it would increase my chances to get selected for GSoC.

Given below is a proof of signing CLA after which I wasn't guided properly despite of me following all the necessary steps.
![image](https://user-images.githubusercontent.com/94903454/163137864-e15435e1-dc63-4cc9-ad20-798cfb95fb8d.png)

Thanks,
Shivam Madlani

**Comment by BenHenning** [comment, marked as answer]
I believe this was now resolved as it seems @seanlip corrected the entry in the CLA table. @ShivamMadlani can you confirm?

/cc @MaskedCarrot

**Reply by MaskedCarrot** [reply]
Yes, Sean corrected Shivam's GitHub handle in the CLA table.

**Reply by ShivamMadlani** [reply]
Yes, the issue has been resolved.
Thanks for the help!
